### PR TITLE
#70 - Fix capitalization and spelling mistakes

### PR DESCRIPTION
**Pull request by daceynolan**
Fix spelling mistakes throughout the docs

closes https://github.com/triggermesh/docs/issues/70

### CODE REVIEW

File: docs/targets/googlesheets.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ ko apply -f config/
 kubectl apply -f samples/
 ```
 
-3. If everything worked sucessfully a new namespace "triggermesh-knative-targets" should be now be created with a Googlesheets adapter and controller pod active and ready!
+3. If everything worked successfully a new namespace "triggermesh-knative-targets" should be now be created with a Google Sheets adapter and controller pod active and ready!
```
Comment (cab105) [inline docs/targets/googlesheets.md:74]: Google Sheets should be GoogleSheets.

File: docs/tm/usage.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ Deploy service from Docker image
 tm deploy service foo -f gcr.io/google-samples/hello-app:1.0 --wait
 ```
 
-If you have Dockerfile for your service, you can use kaniko buildtemplate to deploy it
+If you have Dockerfile for your service, you can use Kaniko build template to deploy it
```
Comment (cab105) [inline docs/tm/usage.md:8]: `buildtemplate` is one word in this context due to the knativism.

File: docs/tm/usage.md [hunk 3]
```diff
@@ -61,7 +61,7 @@ func main() {
 EOF
 ```
 
-Deploy function using Knative lambda buildtemplate with Go runtime
+Deploy function using Knative lambda build template with Go runtime
```
Comment (cab105) [inline docs/tm/usage.md:64]: Same here regarding the `buildtemplate`

**Review by cab105 — APPROVED** [review]
Thank you for the cleanup. Only two minor tweaks.

File: docs/targets/googlesheets.md (hunk 1 above)
Comment (daceynolan) [reply docs/targets/googlesheets.md:74]: @cab105  Thanks for the review. I do not want to be difficult, but when I searched for GoogleSheets in Google, they do have it as two separate words. I am happy to make this change, I just was not sure if there had already been a discussion about the spelling :) 
Screen Shot 2020-11-12 at 9 57 05 AM

File: docs/tm/usage.md (hunk 2 above)
Comment (daceynolan) [reply docs/tm/usage.md:8]: Thanks! I was contemplating this for a while and thought to just hcnage it haha. I will make these changes.

File: docs/targets/googlesheets.md (hunk 1 above)
Comment (cab105) [reply docs/targets/googlesheets.md:74]: Too late. :-P In other cases I do see it references as just Sheets. Feel free to space it out.

Comment (daceynolan) [reply docs/targets/googlesheets.md:74]: haha thanks

**Review by cab105 — APPROVED** [review]

**Pull request merged**